js/components/controlBar.js

```diff
@@ -269,8 +285,9 @@ var ControlBar = React.createClass({
       "oo-control-bar-item oo-live oo-live-indicator": true,
       "oo-live-nonclickable": isLiveNow
     });
-
-    var videoQualityPopover = this.props.controller.state.videoQualityOptions.showVideoQualityPopover ? <Popover><VideoQualityPanel{...this.props} togglePopoverAction={this.toggleQualityPopover} popover={true}/></Popover> : null;
+    var showVideoQualityPanel = !this.props.skinConfig.controlBar.autoplayToggle || this.props.controller.state.configPanelOptions.showVideoQualityPanel;
+    var configPanelContent = showVideoQualityPanel ? <VideoQualityPanel{...this.props} togglePopoverAction={this.toggleConfigPopover} toggleVideoQualityPanel={this.toggleQualityPanel} popover={true}/> :  <ConfigPanel {...this.props} toggleQualityAction={this.toggleQualityPanel} />;
+    var configPanelPopover = this.props.controller.state.configPanelOptions.showConfigPanelPopover  ? <Popover popoverClassName="oo-popover oo-popover-pull-autoplay">{configPanelContent}</Popover> : null;
```

Comment:
Does the name `oo-popover-pull-autoplay` make sense? `oo-popover-pull-right` means it's pulled to the right, right?